src/popup/components/annotationList/AnnotationList.tsx

```diff
@@ -49,15 +54,22 @@ export default class AnnotationList extends React.Component<Partial<IAnnotationL
     } else {
       return (
         <div className={styles.self}>
-          <button onClick={this.handleGoBackClick}>Wróć</button>
-          <ul>
+          <div onClick={this.handleGoBackClick}>
+            <Icon className={styles.chevronButton} icon={ic_chevron_left} size={25}/>
+          </div>
+          <ul className={styles.annotationList}>
           {this.state.annotationLocationData.located.map(annotation => (
             <li
+              className={styles.listItem}
               key={annotation.id}
               data-annotation-id={annotation.id}
-              onClick={this.onAnnotationClick}
+              onClick={() => sendScrollToAnnotation(annotation.id)}
```

Comment:
We decided to stick to a policy of no arrow functions as handlers. We might revise in the future, but for now we stick to it.